controlplane/kubeadm/internal/etcd/util/set.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Modified copy of k8s.io/apimachinery/pkg/util/sets/int64.go
```

Comment:
Add an explanation of what's modified?

Reply:
cc @dlipovetsky 

here is some previous discussion, but this could use further explanation

https://github.com/kubernetes-sigs/cluster-api/pull/2193#discussion_r376550435